src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/ProcessInfo.cs

```diff
@@ -18,7 +19,7 @@ public record ProcessInfo
 
         public int SuccessCode { get; init; }
 
-        public bool Quiet { get; init; }
+        public Func<string, LogLevel> GetMessageLogLevel { get; init; } = msg => msg.Contains("Error") ? LogLevel.Error : LogLevel.Information;
```

Comment:
nit: consider making this function name more specific to tightly couple it to the domain of parsing data from interactive authentication. Our process for parsing log level from other sources may not follow the same rules.

Reply:
This is just a default implementation for how a launched process might decide what log level to use for a message, so I want it to be general.